bot/prompt.py

```diff
@@ -1,16 +1,20 @@
 #!/usr/bin/env python3
 import random
+from typos import add_typos
+from rand_select import random_select_weighted_list
```

Comment:
These should use the local module import format, the same one used in every other file we have, if they're going to be a part of the `bot` module.

```suggestion
from .typos import add_typos
from .rand_select import random_select_weighted_list
```

Reply:
Both `matplotlib` and `seaborn` were just to make sure the rng I was using looked right and are not necessary.